draft-iab-ai-control-report.md

```diff
@@ -23,16 +23,10 @@ author:
  -
     ins: M. Nottingham
     name: Mark Nottingham
-    organization: Cloudflare
-    postal:
-      - Prahran
-    country: Australia
     email: mnot@mnot.net
-    uri: https://www.mnot.net/
```

Comment:
you've removed more than affiliation here...

Reply:
Oops, I read the last few IAB RFCs and thought everything but email should be included. My mistake, I’ll correct it.